modules/promotions/pointers/black-friday.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Elementor\Modules\Promotions\Pointers;
+
+use Elementor\User;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
```

Comment:
I think we can remove this comment :) 
```suggestion
	exit;
```